src/drivers/config.ts

```diff
@@ -22,11 +25,19 @@ export function getTenantConfig(): Config {
   return tenantConfig;
 }
 
-export async function saveConfig(config: Config) {
+export async function saveConfig(orgId: string) {
+  const orgDoc = await getDoc<RawOrgData, object>(
+    bootstrapDoc(`orgs/${orgId}`)
+  );
```

Comment:
Should there be the null check `if (!orgData) throw` here as well?

We wouldn't want to write the config if there was no org found I assume

Reply:
It's there on line 33. We use the bootstrap config if no config was found.